assets/samplesheet.csv

```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
```

Comment:
Need to change the format to look like the one for this pipeline.

Reply:
Actually there is an `assets/samplesheet_pe.csv` and a `assets/samplesheet_se.csv` files created by you, should we just remove this one?